bee-crypto/src/ternary/sponge/kind.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+/// The different kinds of Sponges.
+#[derive(Copy, Clone)]
+pub enum SpongeKind {
+    /// Kerl.
+    Kerl,
+    /// CurlP with 27 rounds.
+    CurlP27,
+    /// CurlP with 81 rounds.
+    CurlP81,
+}
```

Comment:
It probably makes sense to also derive `Debug`, `Eq`, `PartialEq` for this enum.

Reply:
Sure